### PR TITLE
ADDED new question to FAQ

### PR DESCRIPTION
**Pull request by electro199**
ADDED new question to FAQ

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -40,6 +44,23 @@ By default NSFW is disabled. To enable it, you can change the `allow_nsfw` optio
...
+
+This is because you are using python 3.11 to solve this issue delete the python `3.11` and install python `3.10`
+
+## Moivepy error x.mp3 not found
+
+Cause of this error is unknown (possibly due to text leght being longer then limit) but you can try to use other post. 
+
+## Index out of range tried all valid filters 
+
+This happens when bot cant find any post that satisfy the codition try changing `min_comment` `max_comment` etc.
```
Comment (OpenSourceSimon) [inline docs/faq.md]:
```suggestion
This happens when bot can't find any post that satisfy the settings. Try changing `min_comment` or `max_comment`. 
```

File: docs/faq.md [hunk 2]
```diff
@@ -40,6 +44,23 @@ By default NSFW is disabled. To enable it, you can change the `allow_nsfw` optio
...
+
+Cause of this error is unknown (possibly due to text leght being longer then limit) but you can try to use other post. 
+
+## Index out of range tried all valid filters 
+
+This happens when bot cant find any post that satisfy the codition try changing `min_comment` `max_comment` etc. 
+
+## main.py closing automatically
+
+This a is generally cause due to directly opening main.py instead open cmd and run `main.py` it is mentioned in tutorial
```
Comment (OpenSourceSimon) [inline docs/faq.md]:
```suggestion
This a is generally cause due to directly opening main.py by clicking on it. This is the wrong way of running the bot, please open CMD on Windows and run `main.py`. For further information, please check our documentation.
```

File: docs/faq.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
-- [Main.py Not Working](#Main.py-is-not-working)
+- [Main.py Not Working](#mainpy-is-not-working)
 - [Easy Way To Edit Config](#easy-way-to-edit-config)
-- [Read PThe Post Not Just The Title](#some-of-my-markdown-elements-arent-highlighted)
+- [Read The Post Not Just The Title](#read-the-post-not-just-the-title)
 - [TikTok Reader Crashing](#tiktok-reader-crashing)
 - [Preview TTS Voice](#preview-tts-voice)
 - [Could Not Concat Numpy.float64](#could-not-concat-numpyfloat64)
-- [Video Length Is Too Long & I Am Too Lazy To Edit It Myself](#video-length-is-too-long--i-am-too-lazy-to-edit-it-myself)
+- [Video Length Is Too Long & I Am Too Lazy To Edit It Myself](#video-length-is-too-long-i-am-too-lazy-to-edit-it-myself)
```
Comment (OpenSourceSimon) [inline docs/faq.md]: This section was deleted if I remember correctly?
```suggestion
```

**Review by OpenSourceSimon — COMMENTED** [review]
Thanks for the PR!

File: docs/faq.md (hunk 3 above)
Comment (electro199) [reply docs/faq.md]: yeah i think

**Pull request merged**